README.troubleshooting.inference.md

```diff
@@ -0,0 +1,53 @@
+
+## If you see a redis cleanup pod hanging there in the cluster for your deployment... 
+
+it means some redis error has happened during killing your pod. In this case, do:
+
+    kubectl get raycluster -n <xxx> to get the raycluster name for your deployment.
+
+    kubectl edit rayclusters.ray.io -n <xxx> <ray_cluster_name> to bring up the yaml for the deployment.
+
+    Find finalizers: - ray.io/gcs-ft-redis-cleanup-finalizer and delete those 2 lines and save.
+
+    This should apply the change and allow kuberay to clean up the ray cluster without needing the redis cleanup thing to run.
+
+Likely this can be due to your pod name being too long, which translates to your application name being too long. SHORTEN your application name.
+Test Deployment Changes From Local
```

Comment:
I think `Test Deployment Changes From Local` should be a heading as well. It is independent of the first one.